link_shortener/view_routes.py

```diff
@@ -88,7 +88,7 @@ async def owner_specific_links(request, user):
                 ), status=200)
 
 
-@view_blueprint.route('/delete/<link_id>', methods=['GET'])
+@view_blueprint.route('/delete/proceed/<link_id>', methods=['GET'])
```

Comment:
Klidně použij /delete/<id> endpoint na proceed nebo confirm. Je to jednodušší a hezčí endpoint a uživatel stejně nekouká na URL takže zmatečný to snad taky nebude.

Reply:
Jo a kdyby si nechtěl, tak tady taky zvaž to přehození parametrů na /delete/id/endpoint

Reply:
takze takto?
@view_blueprint.route('/delete/<link_id>', methods=['GET'])

Reply:
Jop, takhle je to ideální. Čím jednodušší endpointy, tím lepší. Proto tam občas cpu víc metod (GET, POST) na ten samý endpoint.